client/Windows/wf_graphics.c

```diff
@@ -318,6 +318,14 @@ static BOOL wf_Pointer_SetDefault(rdpContext* context)
 	return TRUE;
 }
 
+static BOOL wf_Pointer_SetPosition(rdpContext* context, UINT32 x, UINT32 y)
+{
+	if (!context)
+		return FALSE;
+
+	return TRUE;
+}
```

Comment:
A good tip, however, this is only a placeholder to prevent crash. So inside it there will be more code. It should probably move a cursor on a client machine appropriately.